doc/api/test.md

```diff
@@ -2965,6 +2969,52 @@ added:
 
 The name of the test.
 
+### `context.plan(count)`
```

Comment:
Can you please mark this as experimental? I don't think we need a warning for this.